docs/mastering-plone/registry.md

```diff
@@ -590,7 +592,7 @@ You can now register these vocabularies as named utilities in {file}`vocabularie
     component="ploneconf.site.vocabularies.talk.RoomsVocabularyFactory" />
 ```
 
-From now on you can use these vocabulary by referring to their name, e.g. `ploneconf.rooms`.
+From now on you can use these vocabulary by referring to their name, for example `ploneconf.rooms`.
```

Comment:
Editorial comment: I always die a little bit inside when I see `e.g.` used without a trailing comma. That or any of its mutations that are not `, e.g.,` between the leading sentence and its following example.

```suggestion
From now on you can use these vocabulary by referring to their name, for example, `ploneconf.rooms`.
```